README.md

```diff
@@ -0,0 +1,67 @@
+<!-- <h1 align="center">The ARINC 853 "CSMIM" Knowledge Base</h1> -->
+<p align="center">
+  <a href="https://github.com/ARINC-IA/CSMIM">
+    <img alt="CSMIM" title="CSMIM" src="docs/csmim.png" width="300">
+  </a>
+</p>
+
+Welcome to the **ARINC 853 Knowledge Base** for the "Cabin Secure Media-Independent Messaging" Protocol.
+This repository contains standardized interface descriptions for aircraft cabin equipment.
+Here, the aerospace industry collaborates to shape the cabin network of the future.
+
+
+## Introduction
+
+This repository complements the ARINC 853 "CSMIM" standard. ARINC 853 specifies a modern
+communication protocol for non-essential cabin equipment. The protocol specification draws
+from related standards in the building and home automation domain, including
+[MQTT](https://mqtt.org/) and [CBOR](https://cbor.io/) as building blocks. Whereas the
+ARINC 853 document describes the generic protocol and data structure, this
+repository &ndash; the CSMIM Knowledge Base &ndash; contains concrete interface definitions for
+various cabin equipment. These definitions are considered "standard" and should be used
+by all cabin equipment suppliers as far as possible.
+
+The goal of our activities is to enable interoperability of all cabin equipment on all
+types of aircraft, and to create an open aircraft cabin network where any supplier can
+add new, useful functions.
+
```

Comment:
should we add a comment that standard definitions can be extended with non-standard properties?

Reply:
I would not water down the goal here in this paragraph.